src/components/GamePanel.vue

```diff
@@ -23,83 +30,72 @@
-          type: "text",
-          id: 0,
-          pairid: 0,
-        } as CardContent,
-        {
-          content:
-            "https://www.shutterstock.com/image-vector/simple-mini-cartoon-ghost-vector-260nw-1470154256.jpg",
-          type: "image",
-          id: 1,
-          pairid: 0,
-        } as CardContent,
-        { content: "some text", type: "text", id: 2, pairid: 0 } as CardContent,
-        {
-          content:
-            "https://www.shutterstock.com/image-vector/simple-mini-cartoon-ghost-vector-260nw-1470154256.jpg",
-          type: "image",
-          id: 3,
-          pairid: 0,
-        } as CardContent,
-        { content: "some text", type: "text", id: 4, pairid: 0 } as CardContent,
-        {
-          content:
-            "https://www.shutterstock.com/image-vector/simple-mini-cartoon-ghost-vector-260nw-1470154256.jpg",
-          type: "image",
-          id: 5,
-          pairid: 0,
-        } as CardContent,
-        {
-          content:
-            "Lorem ipsum dolor sit amet consectetur adipisicing elit. Laborum optio vel sequi. Nam dolorem qui consectetur corrupti quod optio. Libero sequi harum debitis. Quae mollitia aspernatur obcaecati, repellendus eveniet doloribus!",
-          type: "text",
-          id: 6,
-          pairid: 0,
-        } as CardContent,
-        {
-          content:
-            "https://www.shutterstock.com/image-vector/simple-mini-cartoon-ghost-vector-260nw-1470154256.jpg",
-          type: "image",
-          id: 7,
-          pairid: 0,
-        } as CardContent,
-        { content: "some text", type: "text", id: 8, pairid: 0 } as CardContent,
-        {
-          content:
-            "https://www.shutterstock.com/image-vector/simple-mini-cartoon-ghost-vector-260nw-1470154256.jpg",
-          type: "image",
-          id: 9,
-          pairid: 0,
-        } as CardContent,
-        {
-          content: "some text",
-          type: "text",
-          id: 10,
-          pairid: 0,
-        } as CardContent,
-        {
-          content:
-            "https://www.shutterstock.com/image-vector/simple-mini-cartoon-ghost-vector-260nw-1470154256.jpg",
-          type: "image",
-          id: 11,
-          pairid: 0,
-        } as CardContent,
-      ],
+      cards: [] as CardData[],
+      foundPairs: [] as CardPair[],
+      openCardCount: 0,
+      canTurnCard: true,
+      firstCardIndex: -1,
+      disabledCards: [] as boolean[],
+      resetTimeout: 0,
+      allowReset: false,
     };
   },
+  methods: {
+    cardClickProcedure(cardIndex: number) {
+      if (cardIndex == this.firstCardIndex) return;
+      if (this.openCardCount == 0) {
+        this.firstCardIndex = cardIndex;
+      } else if (this.openCardCount == 1) {
+        const card: CardData = this.cards[cardIndex];
+        if (card.pairid == this.cards[this.firstCardIndex].pairid) {
+          console.log("You found a pair!");
```

Comment:
A visual effect for the user will be nice, but I can also understand if you want to make this a separate issue.

Reply:
I will add this in the future